cdk-lib/cloud-demo.ts

```diff
@@ -23,25 +23,40 @@ const env: Environment = {
     region: params.awsRegion
 };
 
+function addTags (stack: cdk.Stack) {
+    const clusterParams = params as prms.ClusterMgmtParams;
+    cdk.Tags.of(stack).add('arkime_cluster', clusterParams.nameCluster);
+
+    if (clusterParams.userConfig.extraTags) {
+        for (const tag of clusterParams.userConfig.extraTags) {
+            cdk.Tags.of(stack).add(tag.key, tag.value);
```

Comment:
Trying to figure out how this works.  If I set some tags initially, then decide I don't want them any more, can I just pass an empty list here and have Cfn remove them on CDK deploy?  In other words - is this setting the tags, or adding them?

Reply:
It just adds, i can clarify in the help text or docs somewhere.